cli/staking/pledge.go

```diff
@@ -49,18 +54,23 @@ var pledgeCmd = &cobra.Command{
 				fmt.Println("Gas spent: ", msg.Receipt.GasUsed)
 			}
 
-			fmt.Println("Message calldata: ", msg.Data)
+			fmt.Println("Message calldata: ", hex.EncodeToString(msg.Data))
 		}
 	},
 }
 
 func init() {
+	pledgeCmd.Flags().StringVarP(&minerAddr, "minerAddr", "m", "", "Storage Provider miner id")
```

Comment:
it is a liitle bit confusing here, becuse param is called minerAddr, and comment says "Storage Provider miner id"
when calling pledge it is only minerId required, so I would consider to change all params to miner id. Because in lotus you can only get minerId for a miner (as far as I know)
Or you can just change comment "Storage provider miner address or id". Should work with both

Reply:
Fixed the CLI flag in the latest commit. Now provides an info of both addresses and actor IDs